sharding/nodesCoordinator/indexHashedNodesCoordinator_test.go

```diff
@@ -1568,6 +1516,14 @@ func TestIndexHashedNodesCoordinator_EpochStart_EligibleSortedAscendingByIndex(t
 					MetachainConsensusGroupSize: 1,
 				}
 			},
+			ChainParametersForEpochCalled: func(_ uint32) (config.ChainParametersByEpochConfig, error) {
```

Comment:
maybe use an assert here to check the epoch is the same provided on L1848/L1849?

Reply:
added